components/dashboard/src/org-admin/MaintenanceNotificationCard.tsx

```diff
@@ -0,0 +1,144 @@
+                type: "success",
+            });
+
+            setIsEditing(false);
+        } catch (error) {
+            console.error("Failed to update maintenance notification message", error);
+            toast.toast({ message: "Failed to update maintenance notification message", type: "error" });
+        }
+    };
+
+    return (
+        <ConfigurationSettingsField>
+            <div className="flex justify-between items-center mb-4">
+                <div>
+                    <Heading3>Scheduled Maintenance Notification</Heading3>
+                    <p className="text-pk-content-tertiary">
+                        Display a notification banner to inform users about upcoming maintenance.
+                    </p>
+                </div>
+                <Button
+                    variant={isNotificationEnabled ? "secondary" : "default"}
+                    onClick={toggleNotification}
+                    disabled={isLoading}
+                >
+                    {isLoading ? "Loading..." : isNotificationEnabled ? "Disable" : "Enable"}
+                </Button>
+            </div>
+
+            {/* Message input section */}
+            <div className="mt-4">
+                <label
+                    htmlFor="maintenance-message"
+                    className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-2"
+                >
+                    Notification Message
+                </label>
+                {isEditing ? (
+                    <div>
+                        <textarea
+                            id="maintenance-message"
+                            className="w-full px-3 py-2 border border-gray-300 dark:border-gray-600 rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 dark:bg-gray-700 dark:text-white"
+                            rows={3}
+                            value={message}
+                            onChange={(e) => setMessage(e.target.value)}
+                            placeholder="Enter a message to display in the notification banner"
+                        />
+                        <div className="mt-2 flex justify-end space-x-2">
+                            <Button
+                                variant="secondary"
+                                onClick={() => {
+                                    setMessage(notificationMessage || DEFAULT_MESSAGE);
+                                    setIsEditing(false);
+                                }}
+                            >
+                                Cancel
+                            </Button>
+                            <Button onClick={saveMessage}>Save</Button>
+                        </div>
+                    </div>
+                ) : (
+                    <div>
+                        <div className="px-3 py-2 border border-gray-300 dark:border-gray-600 rounded-md bg-gray-50 dark:bg-gray-700 text-gray-700 dark:text-gray-300 min-h-[4rem]">
```

Comment:
Probably worth updating to Podkit classes here as well:
```suggestion
                        <div className="px-3 py-2 border border-pk-border-base rounded-md bg-pk-surface-secondary text-pk-content-secondary min-h-[4rem]">
```